src/popup/components/popup-view.tsx

```diff
@@ -192,17 +192,27 @@ export class PopupView extends React.Component<PopupViewProps> {
             <div className="ms-Fabric unsupported-url-info-panel">
                 {this.renderDefaultHeader()}
                 <div className="ms-Grid main-section">
-                    <div
-                        className="launch-panel-general-container"
-                        dangerouslySetInnerHTML={{
-                            __html: DisplayableStrings.urlNotScannable.join('</br>'),
-                        }}
-                    />
+                    <div className="launch-panel-general-container">
+                        <>{this.getUrlNotScannableMessage()}</>
+                    </div>
                 </div>
             </div>
         );
     }
 
+    private getUrlNotScannableMessage = () => {
+        let keyIndex = 0;
+        const result = DisplayableStrings.urlNotScannable.reduce(
+            (r, current) =>
+                r.concat(<span key={keyIndex++}>{current}</span>, <br key={keyIndex++}></br>),
+            [],
+        );
+
+        result.pop();
```

Comment:
I feel like this code could read a bit better; I'm not following why this is necessary? Maybe a comment would help here?

Reply:
I would love to have a better algorithm to do this... but for now, this is the best I could come up with. I'll put a comment on why we need the `.pop()` call just before returning.

Reply:
Done.